wd_helper.au3

```diff
@@ -317,24 +319,26 @@ Func _WD_WaitElement($sSession, $sStrategy, $sSelector, $iDelay = Default, $iTim
 		$sElement = _WD_FindElement($sSession, $sStrategy, $sSelector)
 		$iErr = @error
 
-		If $iErr = $_WD_ERROR_Success Then
-			If $lVisible Then
-				$lIsVisible = _WD_ElementAction($sSession, $sElement, 'displayed')
+		If $iErr = $_WD_ERROR_NoMatch And $b_Check_NoMatch Then
+			Return SetError(__WD_Error($sFuncName, 0), 0, 1)

```

Comment:
When coding I try to not have a bunch of Return points in my functions. Think you can change this so that Return occurs at the bottom with the others?

Reply:
Yes.
I will change this